bin/hyperlog.js

```diff
@@ -135,93 +135,101 @@ const parseStorage = (dir) => {
   return fullPath(dir)
 }
 
+const writeLine = async (path, line) => {
```

Comment:
hmm not sure if we need this since fs has also promises api, afaik we could use fs.promises.writeFile?

Reply:
Do we have any requirements for compatibility with node.js versions? (since fs.promises is supported since v10)

Reply:
anyway .. support for previous versions is ended

Reply:
resolved 32ad836964c8035954bea864bcbb772a88a89255